var/spack/repos/builtin/packages/lapackpp/package.py

```diff
@@ -4,28 +4,46 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+import os
 
 
 class Lapackpp(CMakePackage):
-    """LAPACK++: C++ API for the Basic Linear Algebra Subroutines (University
-    of Tennessee)"""
+    """LAPACK++: C++ API for the LAPACK Linear Algebra Package. Developed
+       by the Innovative Computing Laboratory at the University of Tennessee,
+       Knoxville."""
 
     homepage = "https://bitbucket.org/icl/lapackpp"
-    git      = "https://bitbucket.org/icl/lapackpp"
-    maintainers = ['teonnik', 'Sely85']
+    git = homepage
+    url = 'https://bitbucket.org/icl/lapackpp/downloads/lapackpp-2020.09.00.tar.gz'
+    maintainers = ['teonnik', 'Sely85', 'G-Ragghianti', 'mgates3']
 
-    version('develop', commit="f878fad")
+    version('master', branch='master')
+    version('2020.10.01', sha256='ecd659730b4c3cfb8d2595f9bbb6af65d96b79397db654f17fe045bdfea841c0')
+    version('2020.10.00', sha256='5f6ab3bd3794711818a3a50198efd29571520bf455e13ffa8ba50fa8376d7d1a')
+    version('2020.09.00', sha256='b5d4defa8eb314f21b3788563da9d264e2b084f2eb6535f6c6798ba798a29ee5')
 
-    variant('shared', default=True,
-            description='Build a shared version of the library')
+    variant('shared', default=True, description='Build shared library')
 
+    # Needs to compile against a matching blaspp version
     depends_on('blaspp')
+    depends_on('blaspp@master', when='lapackpp@master')
+    depends_on('blaspp@2020.10.01', when='lapackpp@2020.10.01')
+    depends_on('blaspp@2020.10.00', when='lapackpp@2020.10.00')
+    depends_on('blaspp@2020.09.00', when='lapackpp@2020.09.00')
```

Comment:
Might want to start using a for-loop here since you're repeating the same 4 things here

Reply:
OK done.